time-elements.js

```diff
@@ -561,7 +563,9 @@
     var options = {
       hour: el.getAttribute('hour'),
       minute: el.getAttribute('minute'),
-      second: el.getAttribute('second')
+      second: el.getAttribute('second'),
+      timeZone: el.getAttribute('timeZone'),
+      timeZoneName: el.getAttribute('timeZoneName')
```

Comment:
This custom element is essentially a declarative way of exposing `Intl.DateTimeFormat` to markup, so it might be good to match the attribute names to the key names used by the formatter. In this case, it would be `timeZoneName`, even though that's not quite as nice as `timezone`.